alf/trainers/policy_trainer.py

```diff
@@ -248,6 +248,7 @@ def __init__(self, config: TrainerConfig):
     def initialize(self):
         """Initializes the Trainer."""
         if self._random_seed is not None:
+            os.environ["TF_DETERMINISTIC_OPS"] = str(1)
             random.seed(self._random_seed)
             np.random.seed(self._random_seed)
             tf.random.set_seed(self._random_seed)
```

Comment:
Let's extract the two blocks and move into a e.g. common.set_random_seed(seed) function?

Reply:
Moved